app/assets/stylesheets/frontend/views/_historic-appointments.scss

```
@@ -1,3 +1,15 @@
+.historic-appointments:not(.historical-past-chancellors) .featured-profiles .year-block {
+  @include media(mobile) {
```

I think that this is clearer:

```suggestion
  @include govuk-media-query($until: tablet) {
```